templates/group_vars_mash_servers

```diff
@@ -1932,6 +1937,43 @@ echoip_container_labels_traefik_tls_certResolver: "{{ devture_traefik_certResolv
 # /role-specific:echoip
 
 
+# role-specific:endlessh
+########################################################################
+#                                                                      #
+# endlessh                                                             #
+#                                                                      #
+########################################################################
+
+endlessh_enabled: false
+endlessh_hostname: "{{ mash_playbook_metrics_exposure_hostname }}"
+endlessh_path_prefix: "{{ mash_playbook_metrics_exposure_path_prefix }}/{{ endlessh_identifier }}"
+endlessh_identifier: "{{ mash_playbook_service_identifier_prefix }}endlessh"
+endlessh_base_path: "{{ mash_playbook_base_path }}/{{ mash_playbook_service_base_directory_name_prefix }}endlessh"
+endlessh_uid: "{{ mash_playbook_uid }}"
+endlessh_gid: "{{ mash_playbook_gid }}"
+
+endlessh_container_additional_networks_auto: |
+  {{
+    ([mash_playbook_reverse_proxyable_services_additional_network] if mash_playbook_reverse_proxyable_services_additional_network else [])
+  }}
+
+
+# Only enable Traefik labels if a hostname is set (indicating that this will be exposed publicly)
+endlessh_container_labels_traefik_enabled: "{{ mash_playbook_traefik_labels_enabled and endlessh_hostname }}"
+endlessh_container_labels_traefik_docker_network: "{{ mash_playbook_reverse_proxyable_services_additional_network }}"
+endlessh_container_labels_traefik_entrypoints: "{{ devture_traefik_entrypoint_primary }}"
+endlessh_container_labels_traefik_tls_certResolver: "{{ devture_traefik_certResolver_primary }}"
+
+endlessh_container_labels_metrics_middleware_basic_auth_enabled: "{{ mash_playbook_metrics_exposure_http_basic_auth_enabled }}"
+endlessh_container_labels_metrics_middleware_basic_auth_users: "{{ mash_playbook_metrics_exposure_http_basic_auth_users }}"
```

Comment:
These variables do not seem to exist yet, but.. as mentioned in another comment in this review, separate metrics-related Traefik labels variables should be added, so as to:

- allow the other labels to only handle the web UI
- allow these new variables (and associated labels) to only deal with metrics.. and to allow metrics to be on their own hostname, path prefix and use their own Basic Auth credentials

Reply:
I believe these variables exist in the role's [templates/labels.j2](https://github.com/sudo-Tiz/ansible-role-endlessh/blob/main/templates/labels.j2) and in the role's [default/main](https://github.com/sudo-Tiz/ansible-role-endlessh/blob/main/defaults/main.yml)